sr_grasp/scripts/grasp

```diff
@@ -136,19 +143,18 @@ class GraspNode(object):
                 self.sub_tactile_gazebo = {}
                 for finger in ['lf', 'rf', 'mf', 'ff', 'th']:
                     self.sub_tactile_gazebo[finger] = rospy.Subscriber(
-                        'contacts/' + finger + '/distal', ContactsState,
+                        f'contacts/{finger}/distal', ContactsState,
                         self.gazebo_tactile_cb, finger)
             elif self.hand.get_tactile_type() == 'PST':
                 rospy.loginfo("Using PST tactiles")
                 self.sub_pst = rospy.Subscriber(
                     'tactile', ShadowPST, self.pst_tactile_cb)
             else:
-                rospy.logerr("Don't know how to handle %s tactiles!" %
-                             (self.hand.get_tactile_type()))
+                rospy.logerr(f"Don't know how to handle {self.hand.get_tactile_type()} tactiles!")
 
         rospy.loginfo("Ready to grasp")
 
-    def config_cb(self, config, level):
+    def config_cb(self, config, _level):
```

Comment:
why using a `_` at the beginning of the variable used in your signature? If you don't want to use `level`, AFAIK the convention is `level_`